internal/pkg/circular/circular_test.go

```diff
@@ -85,7 +85,7 @@ func (suite *CircularSuite) TestStreamingReadWriter() {
 
 		p := data
 
-		r := rate.NewLimiter(500000, 1000)
+		r := rate.NewLimiter(300_000, 1000)
```

Comment:
What does the underscore do?

Reply:
nothing, just for readability, I looked at constant and couldn't get how many zeros are there, so I added underscore to make it more readable

Reply:
ahh nice, i didn't know go allows that... pretty neat!